pyaedt/application/aedt_objects.py

```diff
@@ -371,6 +374,9 @@ def oeditor(self):
         if not self._oeditor:
             if self.design_type in ["Circuit Design", "Twin Builder", "Maxwell Circuit", "EMIT"]:
                 self._oeditor = self.odesign.SetActiveEditor("SchematicEditor")
+                if is_linux and settings.aedt_version == "2024.1":
+                    time.sleep(1)
```

Comment:
Is this `time.sleep()` really mandatory?

Reply:
Unfortunately yes, we discussed on how to avoid it and narrowed the scope to specific design types.
However, this is a workaround that we have to use because there are some things that do not work as expected in 2024.1

Reply:
OK. Thanks @SMoraisAnsys for the info.

Reply:
The SetActiveEditor is opening the Design like the SetActiveDesign, so we need to apply the same workaround.